### PR TITLE
Enhancement/swagger doc

### PR DESCRIPTION
**Pull request by wesleymonte**
Made enhancement to the Swagger documentation and added examples for request responses.

### CODE REVIEW

File: src/main/java/org/fogbowcloud/app/jes/arrebol/dtos/QueueDTO.java [hunk 1]
```diff
@@ -2,21 +2,27 @@
...
 
+    @ApiModelProperty(notes = "The Queue Name", position = 2, example = "My Queue")
     public String name;
 
+    @ApiModelProperty(notes = "The number of jobs that are not in Finished state or in Failed state.", position = 3, example = "10")
     @JsonProperty("waiting_jobs")
     @SerializedName("waiting_jobs")
     private long waitingJobs;
 
+    @ApiModelProperty(notes = "The number of hosts used as a worker.", position = 4, example = "10")
```
Comment (thiagomanel) [inline src/main/java/org/fogbowcloud/app/jes/arrebol/dtos/QueueDTO.java:20]: used as workers

**Review by thiagomanel — CHANGES_REQUESTED** [review]
one simple change request

File: src/main/java/org/fogbowcloud/app/jes/arrebol/dtos/QueueDTO.java (hunk 1 above)
Comment (wesleymonte) [reply src/main/java/org/fogbowcloud/app/jes/arrebol/dtos/QueueDTO.java]: Done.

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**